src/app/games/apagos/tests/ApagosDummyMinimax.spec.ts

```diff
@@ -25,4 +28,27 @@ describe('ApagosDummyMinimax', () => {
         const isThereTransfer: boolean = moves.some((move: ApagosMove) => move.isDrop() === false);
         expect(isThereTransfer).toBeFalse();
     });
+    it('should consider having advantage has a higher value board', () => {
+        // Given two states each with more pieces of one specific player
+        const weakerState: ApagosState = ApagosState.fromRepresentation(0, [
+            [0, 0, 0, 0],
+            [7, 0, 0, 1],
+            [7, 5, 3, 1],
+        ], 10, 2);
+        const strongerState: ApagosState = ApagosState.fromRepresentation(0, [
+            [7, 0, 0, 1],
+            [0, 0, 0, 0],
+            [7, 5, 3, 1],
+        ], 2, 10);
+
+        // When computing the value of the boards
+        // Then the state with the most pieces of the player should have a higher value
```

Comment:
J'suis pas sûr d'être convaincu par la formulation du GWT
Given two states with the second having more pieces of player under test in the rightermost space
... Then the second state should be deemed advantageous for the player under test ?

Reply:
Mais mais mais :D ? ET LE WHEN :D ?
Aussi, le titre du test a aucun sens non ?

Reply:
fixed both !